lib/saml_idp/controller.rb

```diff
@@ -35,7 +35,13 @@ def acs_url
 
     def validate_saml_request(raw_saml_request = params[:SAMLRequest])
       decode_request(raw_saml_request)
-      render nothing: true, status: :forbidden unless valid_saml_request?
+      unless valid_saml_request?
+        if Rails::VERSION::MAJOR >= 5
+          head :forbidden
+        else
+          render nothing: true, status: :forbidden
+        end
+      end
```

Comment:
I don't think this conditional is needed. `head` exists in Rails 4 as well: https://github.com/rails/rails/blob/4-2-stable/actionpack/lib/action_controller/metal/head.rb